projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -376,137 +376,6 @@ function wpcom_add_jetpack_menu_item() {
 }
 add_action( 'jetpack_admin_menu', 'wpcom_add_jetpack_menu_item' );
 
-/**
- * Helper function to determine if the admin notice should be shown.
- *
- * @return bool
- */
-function wpcom_site_menu_should_show_notice() {
```

Comment:
Are these changes are intentional 🤔

Reply:
Yes, this notice was displaying the same info as the first step of the new guided tour